pulser-core/pulser/channels/base_channel.py

```diff
@@ -35,6 +35,16 @@
 
 OPTIONAL_ABSTR_CH_FIELDS = ("min_avg_amp",)
 
+States = Literal["u", "d", "r", "g", "h"]  # TODO: add "x" for leakage
+
+STATES_RANK = get_args(States)
+
+EIGENSTATES: dict[str, list[States]] = {
+    "ground-rydberg": ["r", "g"],
+    "digital": ["g", "h"],
+    "XY": ["u", "d"],
+}
```

Comment:
It seems that we are assuming the states here will be always in decreasing order of eigenenergy. I would maybe add a check (an assert is fine) to avoid human error

Reply:
I have added an assert directly in the file, I am not sure that's what you had in mind ?